docker/Dockerfile.nextjs.dev

```
@@ -0,0 +1,80 @@
+    python3 \
+    make \
+    g++ \
+    git \
+    curl
+
+# Enable pnpm via corepack
+RUN corepack enable && \
+    corepack prepare pnpm@latest --activate
+
+# Set pnpm environment variables
+ENV PNPM_HOME="/pnpm" \
```

Hadolint (2.12.0)

[error] Do not refer to an environment variable within the same `ENV` statement where it is defined.

(DL3044)